src/libraries/Microsoft.PowerFx.Core/Utils/DPath.cs

```diff
@@ -11,7 +11,10 @@
 
 namespace Microsoft.PowerFx.Core.Utils
 {
-    // A path is essentially a list of simple names, starting at "root".
+    /// <summary>
+    /// A list of simple names (<see cref="DName" />), starting at "root" (<see cref="Root" />).
+    /// </summary>
+    /// // TASK: 67008 - Make this public, or expose a public shim in Document.
```

Comment:
nit: remove this, you're making it public so the ancient todo is no longer needed